### PR TITLE
Included a stress test for Kruize remote monitoring mode

### PR DESCRIPTION
**Pull request by chandrams**
Included a stress test for Kruize remote monitoring mode

### CODE REVIEW

**Comment by dinogun** [comment]
Can you please add some documentation on this stress test and how to invoke it?

**Comment by chandrams** [comment]
@dinogun - I have included the README, please review. Thanks.

File: tests/scripts/remote_monitoring_stress_test/README.md
Comment (dinogun) [inline tests/scripts/remote_monitoring_stress_test/README.md:1]: @chandrams can you move this folder under `remote_monitoring` and rename this as `stress_test`. You can have a README inside `remote_monitoring` that points to this.
Also please add a link to the top level README.md inside tests to the README inside `remote_monitoring`

**Review by dinogun — APPROVED** [review]
Awesome. LGTM

**Comment by dinogun** [comment]
@chandrams I see all of the earlier commits are not signed. Please make sure to sign all of them. Maybe we need to catch this as a PR check

**Pull request merged**